<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultForm</class>
 <widget class="QDialog" name="ResultForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Здоровье в цифрах</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/main/img/logo.jpg</normaloff>:/main/img/logo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="resultFormFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>601</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="bmrOutput">
       <property name="text">
        <string>bmrOutput</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="bmiOutput">
       <property name="text">
        <string>bmiOutput</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="carbsOutput">
       <property name="text">
        <string>carbsOutput</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="fatsOutput">
       <property name="text">
        <string>fatsOutput</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="proteinsOutput">
       <property name="text">
        <string>proteinsOutput</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="bmiLabel">
       <property name="text">
        <string>Ваш ИМТ:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="bmrLabel">
       <property name="text">
        <string>Суточная норма калорий:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="proteinsLabel">
       <property name="text">
        <string>Белки:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="fatsLabel">
       <property name="text">
        <string>Жиры:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="carbsLabel">
       <property name="text">
        <string>Углеводы:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
